### DISCUSSION TITLE
Rcom support for Ubuntu 22.04.2 LTS version and kernal - 5.19.0-41

### DISCUSSION THREAD

**Discussion by mathanprasannakumar** [ROCm/ROCm#2127] [Q&A] [answered]
I want to know is Rcom is Compatable for Ubuntu 22.04.2 LTS version , 

As some of the Rcom versions are compatible withe 22.04.1 and 22.04 , i am having a doubt whether this is compatible with 22.04.2 or should i need wait for the suitable release ?

**Comment by xuhuisheng** [comment]
As https://docs.amd.com/bundle/ROCm-Release-Notes-v5.5/page/About_This_Document.html, ROCm-5.5.0 can support ubuntu-22.04.2 with kernel-5.19.

**Reply by mathanprasannakumar** [reply]
That really great , one more i need to know is Rcom is compatible with RX5500m , I found that i am not having the GPUs mentioned in the documention, is any other versions of Rcom that is compatible for RX 5500M  as i am ready to switch any distro supporting such Rcom.

**Reply by xuhuisheng** [reply, marked as answer]
I am afraid ROCm won't support RX5500m officially.

**Reply by mathanprasannakumar** [reply]
Thankyou for your time , i also spent time looking for it , no support for RX5500m .
